src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Formatters/PackageDeprecationMetadataContextInfoFormatter.cs

```diff
@@ -46,8 +46,6 @@ private PackageDeprecationMetadataContextInfoFormatter()
                 }
             }
 
-            Assumes.NotNullOrEmpty(message);
```

Comment:
So this is the actual change, right? We had no way of seeing this failed assumption... unless it was in the ActivityLog, maybe?

Reply:
That is the change, yes.  This was failing when we tried to de-serialize the data coming back from the server, I am not sure where the exception actually ended up, but it just resulted in us not getting this data.